.github/workflows/rust.yml

```diff
@@ -198,64 +198,12 @@ jobs:
       - name: Check uDeps
         run: cargo udeps --all-targets
 
-  fuzz-translate:
-    name: Fuzz (Translation)
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332
-        with:
-          submodules: true
-      - uses: dtolnay/rust-toolchain@master
-        with:
-          # Pin nightly so that it does not invalidate GitHub Actions cache too frequently.
-          toolchain: nightly-2024-09-24
-      - name: Set up Rust cache
-        uses: Swatinem/rust-cache@v2
-        with:
-          cache-directories: |
-            ~/fuzz/corpus/translate/
-            ~/fuzz/corpus/translate_metered/
-      - name: Install cargo-fuzz
-        run: |
-          # Note: We use `|| true` because cargo install returns an error
-          #       if cargo-udeps was already installed on the CI runner.
-          cargo install cargo-fuzz || true
-      - name: Build Fuzzing
-        run: cargo fuzz build translate
-      - name: Fuzz (Translation)
-        run: cargo fuzz run translate -j 2 --verbose -- -max_total_time=60 # 1 minute of fuzzing
-      - name: Fuzz (Translation) + fuel
-        run: cargo fuzz run translate_metered -j 2 --verbose -- -max_total_time=60 # 1 minute of fuzzing
-
-  fuzz-execute:
-    name: Fuzz (Execution)
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332
-        with:
-          submodules: true
-      - uses: dtolnay/rust-toolchain@master
-        with:
-          # Pin nightly so that it does not invalidate GitHub Actions cache too frequently.
-          toolchain: nightly-2024-09-24
-      - name: Set up Rust cache
-        uses: Swatinem/rust-cache@v2
-        with:
-          cache-directories: |
-            ~/fuzz/corpus/execute/
-      - name: Install cargo-fuzz
-        run: |
-          # Note: We use `|| true` because cargo install returns an error
-          #       if cargo-udeps was already installed on the CI runner.
-          cargo install cargo-fuzz || true
-      - name: Build Fuzzing
-        run: cargo fuzz build execute
-      - name: Fuzz (Execution)
-        run: cargo fuzz run execute -j 2 --verbose -- -max_total_time=120 # 2 minutes of fuzzing
-
-  fuzz-differential:
-    name: Fuzz (Differential)
+  fuzz:
```

Comment:
Do we have just one `fuzz` job because fuzzing now runs everything multi-threaded? I split up the fuzzing jobs in the past because that allowed us to use more CI core at the same time since each CI machine comes with 2 cores. So 3 jobs -> 6 CPU cores for fuzzing. With this it seems we are back to just 2 cores. Is that correct? If yes, could we fix this in an elegant way? Obviously using 3 jobs was kind of a "hack".

Reply:
This is still 4 jobs for the 4 fuzzing targets. A job runs for each variable in `strategy.matrix`.

https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix